Packs/ApiModules/Scripts/AggregatedCommandApiModule/AggregatedCommandApiModule.py

```diff
@@ -442,24 +508,46 @@ def create_indicator(self) -> list[dict]:
             list[dict]: The final list structure.
         """
         results: list[dict] = []
-        for indicator_value, tim_context_result in self.tim_context.items():
-            current_indicator: dict[str, Any] = {"Value": indicator_value}
-            if tim_indicator := [indicator for indicator in tim_context_result if indicator.get("Brand") == "TIM"]:
-                if self.indicator.type == "file":
-                    hashes_dict = self.indicator.get_all_values_from(tim_indicator[0]) or build_hash_dict(indicator_value)
-                    current_indicator.update({"Hashes": hashes_dict})
-                current_indicator.update(
-                    {
-                        "Status": pop_dict_value(tim_indicator[0], "Status"),
-                        "ModifiedTime": pop_dict_value(tim_indicator[0], "ModifiedTime"),
-                    }
+        for indicator_instance in self.indicator_instances:
```

Comment:
This function is really hard to follow on - let's split this as well into helper functions. 
The create should only look something like this:
```

    for instance in self.indicator_instances:
        instance.compute_status()

        if instance.final_status == Status.FAILURE:
            results.append(self._build_failure_indicator(instance))
            continue

        results.append(self._build_success_indicator(instance))

    return results
```